### ISSUE TITLE
Multiple warnings when running the big data workflow

### ISSUE THREAD

**Issue by johnchase** [benjjneb/dada2#109]
I am following the [big data workflow](http://benjjneb.github.io/dada2/bigdata.html). The following code results in multiple warning messages. I don't have a deep enough understanding of dada2 to know if these warnings are safe to ignore, of if changes to creating the error model need to be made, or what they are actually indicating. Any clarification would be greatly appreciated. Thanks!

```{r}
dds <- vector("list", length(sample.names))
names(dds) <- sample.names
for(sam in sample.names) {
  cat("Processing:", sam, "\n")
    derep <- derepFastq(filts[[sam]])
    dds[[sam]] <- dada(derep, err=err, multithread=TRUE)
}
rm(derep)
```

```{r}
Processing: sample1.fastq 
Sample 1 - 7630 reads in 2768 unique sequences.

Processing: sample2.fastq 
Sample 1 - 2 reads in 2 unique sequences.
Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : at  13.875Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : radius  446.27Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : all data on boundary of neighborhood. make span biggerWarning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : pseudoinverse used at 13.875Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : neighborhood radius 21.125Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : reciprocal condition number  1Warning message:
In simpleLoess(y, x, w, span, degree = degree, parametric = parametric, : There are other near singularities as well.
```

**Comment by benjjneb** [comment]
Should be ignored.

Basically it is complaining that it can't really estimate the error rates from 2 reads in sample2.fastq. But you aren't estimating the error rates here, you've done that previously.

I should do something about suppressing those warnings in this case though. It looks scary.

**Comment by benjjneb** [comment]
Fixed in f320d14ff97cf6ff39b2a781740265246d86b68d

(`loess` warnings suppressed when `selfConsist=FALSE`)

**Comment by johnchase** [comment]
Perfect, thanks!
